README.md

```diff
@@ -1,2 +1,186 @@
-# Awesome-books
-website to get list of books
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!--   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+<!--   <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🔬 Awesome-books<a name="about-project"></a>
+
+This project is for an list of awesome books in a webpage
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+  </ul>
+  <ul>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- a collection that keeps a list of books
+- new book is added to the collection and desplayed in the webpage when add button clicked
+- book is removed from the collection and dissapears from the page when remove button is clicked
+- data is preserved in the browser's memory by using localStorage
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [OPTIONAL] It will be nice to update the `Getting Started` section of your readme file with explicit steps on how to clone and run you app locally as this will be of help to non technical users and some of them are potential recruiters.